avocado/plugins/spawners/remote.py

```diff
@@ -161,7 +161,7 @@ async def spawn_task(self, runtime_task):
                 )
                 return False
 
-        status, output = await RemoteSpawner.run_remote_cmd_async(session, shlex.join(entry_point_args), 3600)
+        status, output = await RemoteSpawner.run_remote_cmd_async(session, shlex.join(entry_point_args), 14400)
```

Comment:
Shouldn't this cooperate with the test's timeout?  What if the test's has a larger timeout than the one hardcoded here?

Reply:
No this is in fact a very good question and I was anticipating it. My question about the above is the following: is it reasonable to set some configurable test timeout for a spawner? IIUC the process, podman, as well as LXC spawners are all starting a command/subprocess with some isolation and keep it unlimited in terms of execution time. However, in aexpect I had to provide a timeout for the session command and it might be sane to enforce test tasks to complete eventually. So perhaps for now I could search for a way to either not enforce any timeout or enforce some extremely large (for most practical reasons) timeout. Or ultimately depending on your answer above we might also enforce a general timeout for any spawner and thus spawned task.

Reply:
I see you removed the timeout, which I believe was the best thing to do.  Marking as resolved.